### PR TITLE
Updated with information ready for Heroku

### PR DESCRIPTION
**Pull request by veerleprins**
I've updated the pages to get index.ejs as home i.s.o. /home.ejs and changed the PORT for Heroku (with Procfile)

### CODE REVIEW

File: routes/route.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ async function home(req, res, next) {
     req.session.gender = myself[0].prefGender;
     req.session.movie = myself[0].prefMovie;
     let filtered = await checkGenderPref(allUsers, myself);
-    res.render('home.ejs', {
+    res.render('index.ejs', {
```
Comment (rowanhorn1412) [inline routes/route.js:232]: what is inside of the index.ejs?

Comment (joordy) [reply routes/route.js:232]: The homescreen! Cause we don't need Home anymore

File: view/404.ejs [hunk 2]
```diff
@@ -13,7 +13,7 @@
             <h2>Ooooops</h2>
             <img src="images/icons/error.svg" alt="Warning sign">
             <p>Sorry, the page you are trying to reach does not exist...Go back to the home page.</p>
-            <a href="/home">Go to homepage</a>
+            <a href="/">Go to homepage</a>
```
Comment (joordy) [inline view/404.ejs:16]: Did you do al the components also? Otherwise, I'll have a look at it.

Comment (veerleprins) [reply view/404.ejs:16]: Oh shit, I forgot the components, oops! Yes, if you would take a look at that, thanks! 😸

**Pull request merged**